dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1606,18 +1606,23 @@ a.download-video {
   }
 }
 
+.level-group-number {
+  display: block;
+  font-size: 16px;
+  line-height: 24px;
+  float: left;
+  width: 30px;
+}
+
+.level-group-content {
+  overflow: hidden;
```

Comment:
Might make sense to have .level-group { .number {} .content {} } except I worry about naming collisions, so don't worry about that for now :)